src/main/java/fr/uge/lp/databackbonepoc/linkresearch/sampler/BasicSampler.java

```diff
@@ -0,0 +1,19 @@
+package fr.uge.lp.databackbonepoc.linkresearch.sampler;
+
+import java.util.stream.Stream;
+
+public final class BasicSampler implements Sampler {

```

Comment:
```suggestion
public final class DeterministicSampler implements Sampler {
```

Et du coup ça fait doublon avec celui que Perrine avait fait.